src/components/Widgets/Widgets.style.tsx

```diff
@@ -17,7 +17,7 @@ export const WidgetContainer = styled(Box, {
     display: 'flex',
     margin: '0 auto 24px',
     overflow: !welcomeScreenClosed ? 'hidden' : 'inherit',
-    width: '100%',
+    width: 'auto',
```

Comment:
why do you have to put back to 'auto'? It make some themes bug.

Reply:
so i have put the navbar next to the widget (wrapped together within `<Stack>` component) and when widget takes 100% width, then the navbar is being pushed out of the screen. If widget has to have 100% of the width, then the navbar has to be positioned absolute i guess